server/src/test/proto/spine/test/stand/cafeteria.proto

```diff
@@ -20,6 +20,8 @@ message Menu {
 
     MenuId id = 1;
     repeated Dish dish = 2;
```

Comment:
Can you please either have empty lines between all three fields, or no empty lines at all.